apps/monk-test-app/package.json

```diff
@@ -53,11 +59,7 @@
     "eslint-plugin-promise": "^6.1.1",
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-utils": "^3.0.0",
-    "jest": "^29.3.1",
-    "prettier": "^2.7.1",
-    "regexpp": "^3.2.0",
-    "ts-jest": "^29.0.3"
+    "eslint-utils": "^3.0.0"
```

Comment:
Why are all these dependencies declared in the "peerDependencies" by the way? I think they should be decalred in the regular dependencies

Reply:
Yes, we should defined these dependencies as **"peerDependencies"** in test application. As project throws following errors, if place them into regular dependencies:
* Application build gets failed for React-Native demo test app as **"@react-native-community"** internally uses ESLint packages and demand them from project's **"node_modules"** folder itself
* ESLint error when using ESLint and TypeScript dependencies as regular dependencies. Sharing one error message for the reference:
```
ESLint couldn't determine the plugin "jsx-a11y" uniquely

/Users/rohit.sharma/workspace/projects/ACV/monkjs_refacto/node_modules/eslint-plugin-jsx-a11y/lib/index.js (loaded in ".eslintrc.js » @monkvision/eslint-config-typescript-react-native")

/Users/rohit.sharma/workspace/projects/ACV/monkjs_refacto/apps/monk-test-app-native/node_modules/eslint-plugin-jsx-a11y/lib/index.js (loaded in ".eslintrc.js » @monkvision/eslint-config-typescript-react-native » plugin:jsx-a11y/recommended")
```

Reply:
This type of fix might require sometime for research. So we can take care of this rectification later.